SDWebImage/SDWebImageDownloaderOperation.m

```diff
@@ -344,12 +344,12 @@ - (void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)data
     }
     
     // Get the finish status
-    BOOL finished = (self.receivedSize >= self.expectedSize);
+    BOOL downloadFinished = (self.receivedSize >= self.expectedSize);
     // Get the current progress
     double currentProgress = (double)self.receivedSize / (double)self.expectedSize;
     double previousProgress = self.previousProgress;
     // Check if we need callback progress
-    if (currentProgress - previousProgress < self.minimumProgressInterval) {
+    if (!downloadFinished && currentProgress - previousProgress < self.minimumProgressInterval) {
```

Comment:
Just a small suggestion, can you add the parenthesis at `currentProgress - previousProgress`? just like `!downloadFinished && (currentProgress - previousProgress) < self.minimumProgressInterval`. Maybe it's more readable.

Reply:
I'll do that.